conode/conode.go

```diff
@@ -129,6 +149,48 @@ func setup(c *cli.Context) error {
 	if c.String("debug") != "" {
 		log.Fatal("[-] Debug option cannot be used for the 'setup' command")
 	}
+
+	if c.Bool("batch") {
+		port := c.Int("port")
+		portStr := fmt.Sprintf("%v", port)
+
+		serverBinding := network.NewAddress(network.TLS, net.JoinHostPort("", portStr))
+		kp := key.NewKeyPair(cothority.Suite)
+
+		pub, _ := encoding.PointToStringHex(cothority.Suite, kp.Public)
+		priv, _ := encoding.ScalarToStringHex(cothority.Suite, kp.Private)
+
+		conf := &app.CothorityConfig{
+			Suite:       cothority.Suite.String(),
+			Public:      pub,
+			Private:     priv,
+			Address:     serverBinding,
+			Description: c.String("description"),
+		}
+
+		dir := cfgpath.GetConfigPath("conode")
+		out := path.Join(dir, "private.toml")
+		err := conf.Save(out)
```

Comment:
Are you sure that `app.InteractiveConfig` won't overwrite this file? I had a quick look at `app.InteractiveConfig` and didn't see anything that checks for existing configs.